ipynb/examples/utils/testenv_example.ipynb

```diff
@@ -105,135 +200,92 @@
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "05:59:27  INFO    :         Target - Using base path: /home/derkling/Code/lisa\n",
```

Comment:
(As per f2f discussion) I think we should update the output along with the code cells. I think we just have to live with the fact that looking at diffs of ipynb files is 0 fun.